### PR TITLE
[Behat] Changed the way of how the "I am on the page" assertion works

### PR DESCRIPTION
**Pull request by pamil**
Using PHP 5.5 generators (based on #3110), "I am on the (.+)? (page|step)" assertion compares current route with expected one, instead of comparing current address with expected address (generated from expected route). What it means - routes with parameters will work like a charm :)

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,16 +19,16 @@
         }
     ],
     "require": {
-        "php":                                  ">=5.3.9",
+        "php":                                  "~5.5",
         "a2lix/translation-form-bundle":        "~2.0",
         "doctrine/collections":                 "~1.2",
-        "doctrine/common":                      "~2.3",
+        "doctrine/common":                      "~2.3,<2.5",
```
Comment (stloyd) [inline composer.json]: Why?

Comment (pjedrzejewski) [inline composer.json]: There is a bug in 2.5.1 IIRC. @pamil Right?

Comment (pamil) [inline composer.json]: Version `2.5.0` has security issue and version `2.5.1` fixes that issue but introduces some bugs that affects our `@products` suite.

I think we may set `2.5` as default in `0.16`, but it'll need more work.

Comment (pamil) [inline composer.json]: By the way, it is code from another PR, #3110 :)

**Comment by pamil** [comment]
@pjedrzejewski ping, want to clean dependencies of #3134, as it depends on 3 pull requests now :)

**Comment by pamil** [comment]
Closing as it's impossible to get page route while using Selenium2 driver and now we got javascript suite running.

**Pull request closed without merging**